centreon/src/Core/Dashboard/Infrastructure/Repository/DbWriteDashboardRepository.php

```diff
@@ -133,13 +137,15 @@ public function update(Dashboard $dashboard): void
     private function bindValueOfDashboard(\PDOStatement $statement, Dashboard|NewDashboard $dashboard): void
     {
         $statement->bindValue(':name', $dashboard->getName());
-        $statement->bindValue(':description', $this->emptyStringAsNull($dashboard->getDescription()));
+        $statement->bindValue(':description', $dashboard->getDescription());
```

Comment:
Again why ? This does not have the same behavior

Reply:
Description should be handled as it is, a string or a nullable, there is actually no need to cast an empty string as null.